ReactiveXComponent/Properties/AssemblyInfo.cs

```diff
@@ -1,5 +1,4 @@
-﻿using System.Reflection;
```

Comment:
Why do you want to keep this file instead of moving everything in the csproj ?

Reply:
Yes i just saw this when i want to add the old info of .nuspec but if we want to get the version with the cake script we can keep this but we can also do in .csproj. tell me what you want to do exactly

Reply:
i remove the assembly info and add info in ReactiveXComponent.csproj